docs/index.rst

```diff
@@ -6,11 +6,6 @@
 PlasmaPy Documentation
 ######################
 
-.. toctree::
-   :hidden:
-
-   /auto_examples/plot_distribution
-
```

Comment:
I removed these since they made the examples appear in the table of contents menu at the left side ("Braginskii coefficients" etc here):

![image](https://user-images.githubusercontent.com/11289391/60425424-c6ba4d80-9bf2-11e9-9fca-76aa0ed88c83.png)